.devcontainer/devcontainer.json

```diff
@@ -3,6 +3,5 @@
     "build": {
       "dockerfile": "Dockerfile"
     },
-    "extensions": ["Iterative.dvc"],
-    "postCreateCommand": "pip3 install --user -r requirements.txt"
```

Comment:
[F] There were two issues here.

1. The extension was installed before the packages are installed globally.
2. The Python extension was not installed so we could not hook into the default interpreter that is selected.

When the packages are installed inside of a virtual environment this triggers a file watcher that then tries the CLI to see if it is accessible. We cannot do this when DVC is installed as a global module because we would need to watch the various Python package directories to see if updates have been made (the directories are platform-specific for a start).

Reply:
Probably want to use the postCreateCommand to pull the data for this repo contents:

![image](https://user-images.githubusercontent.com/37993418/176331289-38ec15ce-0b7c-4003-b2ce-ddbbb0c7f14b.png)

i.e `"postCreateCommand": "python3 -m dvc pull"`